TPP.Persistence.MongoDB/Repos/UserRepo.cs

```diff
@@ -44,17 +47,26 @@ static UserRepo()
                 cm.MapProperty(u => u.GlowColor).SetElementName("secondary_color");
                 cm.MapProperty(u => u.GlowColorUnlocked).SetElementName("secondary_color_unlocked");
                 cm.MapProperty(u => u.PokeyenBetRank).SetElementName("pokeyen_bet_rank");
-                cm.MapProperty(u => u.Roles).SetElementName("roles");
+                cm.MapProperty(u => u.Roles).SetElementName("roles")
+                .SetDefaultValue(new HashSet<Role>());
             });
         }
 
-        public UserRepo(IMongoDatabase database, long startingPokeyen, long startingTokens)
+        public UserRepo(IMongoDatabase database, long startingPokeyen, long startingTokens, IImmutableList<string> defaultOperators)
         {
             database.CreateCollectionIfNotExists(CollectionName).Wait();
             Collection = database.GetCollection<User>(CollectionName);
             _startingPokeyen = startingPokeyen;
             _startingTokens = startingTokens;
+            _defaultOperators = defaultOperators.ToImmutableHashSet<string>();
             InitIndexes();
+
+            foreach (string name in _defaultOperators)
+            {
+                User? user = FindBySimpleName(name.ToLower()).Result;
+                if (user != null)
+                    SetRoles(user, new HashSet<Role> { Role.Operator });
```

Comment:
this looks like it also removes any existing roles

Reply:
It was written with the assumption that operator is a sort of "master role" and wouldn't need (or make sense to have) other roles. This can be changed.

Reply:
hmm you have a point. I'm indifferent about this now